docs/proposals/project-scoped-repository-enhancements.md

```diff
@@ -53,8 +53,11 @@ credentials. In the first case, we need to maintain backwards compatibility for
 is that the API will return the first repository found matching the URL given. Since we now want to allow the same URL 
 to potentially be in multiple projects, we need to do some minor changes.
 
-* If there is only one matching repository with the same URL and assuming the user is allowed to access it, use that repository 
-whether it is project-scoped or not. This is in line with the current behavior.
+* If there is only one matching repository with the same URL, and assuming the user is allowed to access it _and_ there is
+no app project given as a parameter, use that repository whether it is project-scoped or not. This is in line with the 
```

Comment:
I'm not a fan of `whether it is ... or not` -- my inclination is to write `whether or not ...`, but that doesn't improve it much. My next iteration favors `regardless`, but writing that out was awkward...

how about something along these lines:
```suggestion
no app project given as a parameter, use that repository ignoring any project-scope. This is in line with the 
```